clients/ui/bff/internal/integrations/httpclient/http.go

```diff
@@ -120,7 +120,11 @@ func (c *HTTPClient) POST(url string, body io.Reader) ([]byte, error) {
 		return nil, err
 	}
 
-	req.Header.Set("Content-Type", "application/json")
+	ct := "application/json"
```

Comment:
Also, I suggest having two versions of this method. one that we don't pass content type (and we use "application/json as a parameter) and another one that sets it by default. Would that fit your needs?